ios/PurchasesHybridCommon/PurchasesHybridCommon/ErrorContainer.swift

```diff
@@ -27,9 +27,10 @@ import Purchases
 
         info["underlyingErrorMessage"] = underlyingErrorMessage ?? ""
 
-        if let readableErrorCode = nsError.userInfo[Purchases.ReadableErrorCodeKey] {
+        // todo: see about exposing "readable_error_code" from purchases-ios
```

Comment:
this is no longer exposed directly, so I'm just hardcoding the string for now. We should expose it and use the value directly.

Reply:
that's a good idea. To be honest I'm a bit scared about making more changes here because I'm not 100% sure how these are used, but perhaps we can do another pass at these before shipping the new version?